recipes/ca-policy-lcg/meta.yaml

```diff
@@ -0,0 +1,319 @@
+    sha256: d85dabee85181a40bff503cde5f758008fe021de90d03a7abdb9fb027a0cc5c5
+    folder: ca-policy-lcg
+  - url: https://repository.egi.eu/sw/production/cas/1/current/tgz/ca_TERENA-eScience-SSL-CA-3-{{ version }}.tar.gz
+    sha256: 6b50ee3301da470e61efdb4f6a2e40fc3d70e145db629893362877ffa6407c0d
+    folder: ca-policy-lcg
+  - url: https://repository.egi.eu/sw/production/cas/1/current/tgz/ca_TERENAeSciencePersonalCA3-{{ version }}.tar.gz
+    sha256: a7f337dec32d21e42b85d20345fe6274e761dce0c2d0a6c1317624e2c98dfc53
+    folder: ca-policy-lcg
+  - url: https://repository.egi.eu/sw/production/cas/1/current/tgz/ca_TRGrid-{{ version }}.tar.gz
+    sha256: 831bdd9d732a52c4ad3695e232d0d9d8d8603aabaea4aab676745e661c7c5cb9
+    folder: ca-policy-lcg
+  - url: https://repository.egi.eu/sw/production/cas/1/current/tgz/ca_TSU-GE-{{ version }}.tar.gz
+    sha256: 5b0d0ee7cfa47a2cac081c12ed93cd0b48d28fe8b3fef858dd9eefa9ad333b5b
+    folder: ca-policy-lcg
+  - url: https://repository.egi.eu/sw/production/cas/1/current/tgz/ca_UGRID-G2-{{ version }}.tar.gz
+    sha256: d76e63eb9d36e89a4aad124721ea29d10b40923683edd0222da7f85b585c6995
+    folder: ca-policy-lcg
+  - url: https://repository.egi.eu/sw/production/cas/1/current/tgz/ca_UKeScienceCA-2B-{{ version }}.tar.gz
+    sha256: 16d36d16c94f071dd7e77efe6c42f6559e62033d547735f2a492792277430a71
+    folder: ca-policy-lcg
+  - url: https://repository.egi.eu/sw/production/cas/1/current/tgz/ca_UKeScienceRoot-2007-{{ version }}.tar.gz
+    sha256: 0852a8d4453cee81567ab98ac00613a1869fc63556cb67ab4b9470fb3a8a077b
+    folder: ca-policy-lcg
+  - url: https://repository.egi.eu/sw/production/cas/1/current/tgz/ca_UNAMgrid-ca-{{ version }}.tar.gz
+    sha256: f2aa086bda43f8d462d0532d31a36a29cb12b23a7c5325c980ed6eb0bbcbe9eb
+    folder: ca-policy-lcg
+  - url: https://repository.egi.eu/sw/production/cas/1/current/tgz/ca_UNLPGrid-{{ version }}.tar.gz
+    sha256: 159a8cfb2ac4e2130342673306b4fb2d280238f47e666d0f80ca01251261ca9e
+    folder: ca-policy-lcg
+  - url: https://repository.egi.eu/sw/production/cas/1/current/tgz/ca_cilogon-osg-{{ version }}.tar.gz
+    sha256: 2656c42f7d1025002c92ee6415699033bb0f2bcaf44e153a04fca8719362e53d
+    folder: ca-policy-lcg
+  - url: https://repository.egi.eu/sw/production/cas/1/current/tgz/ca_cilogon-silver-{{ version }}.tar.gz
+    sha256: c326734e8afbfb101bcdd4369fbe1afaf33ffee2ba634cca63d528bed35d758c
+    folder: ca-policy-lcg
+  - url: https://repository.egi.eu/sw/production/cas/1/current/tgz/ca_seegrid-ca-2013-{{ version }}.tar.gz
+    sha256: 42d446612aa98b76d9120d5a64a6204a253c5ddba3ffd57eb46c187d60cd52d2
+    folder: ca-policy-lcg
+
+build:
+  noarch: generic
+  number: 0
+  # DIRAC will try to update these automatically so avoid linking between environments
+  no_link:
+    - ${PREFIX}/etc/grid-security/certificates/*
+
+requirements:
+  build:
+    # Dummpy requirement to ensure conda-forge builds this correctly
```

Comment:
I've opened an issue about this (https://github.com/conda-forge/staged-recipes/issues/8377)